### PR TITLE
move all test-infra jobs onto podutils

### PR DESCRIPTION
**Pull request by krzyzacy**
how ~~confident~~ lucky am I

/hold
/assign @cjwagner @stevekuznetsov @fejta @spiffxp @BenTheElder

### CODE REVIEW

File: config/jobs/kubernetes/test-infra/janitors.yaml [hunk 1]
```diff
@@ -82,15 +74,12 @@ periodics:
...
     containers:
-    - args:
-      - --bare
-      - --timeout=60
-      - --service-account=/etc/service-account/service-account.json
-      - --upload=gs://kubernetes-jenkins/logs
-      - --scenario=kubernetes_janitor
-      - --
+    - command:
+      - ./workspace/scenarios/kubernetes_janitor.py
```
[the remark this answers is not in the document]
Comment (krzyzacy) [reply config/jobs/kubernetes/test-infra/janitors.yaml]: scenarios are built into bootstrap image now (https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/Dockerfile#L127-L131), and there should be no other test-infra dependencies for these jobs.

**Comment by stevekuznetsov** [comment]
Nice!

**Comment by krzyzacy** [comment]
rebased

File: config/jobs/kubernetes/test-infra/janitors.yaml [hunk 2]
```diff
@@ -1,19 +1,14 @@
 periodics:
 - interval: 1h
   name: maintenance-ci-aws-janitor
```
Comment (fejta) [inline config/jobs/kubernetes/test-infra/janitors.yaml]: I veto LGTMing this until it is deleted and/or passing.

**Comment by krzyzacy** [comment]
cc @amwat 
things may or may not pass, if not, then happy holidays!

/hold cancel

**Pull request merged**